lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -118,6 +118,7 @@ def __init__(self,
                  *,
                  config,
                  allocator_type,
+                 static_ip_enabled=False,
```

Comment:
this constructor param isn't useful and leaks an internal detail that should not be known by this component. there are a ton of other problems with this constructor but outside of the scope of this change so just nuke this param for now